src/main/scripts/jenkins

```diff
@@ -2,33 +2,66 @@
 
 set -e
 
-nprocs="$(grep -c '^processor' /proc/cpuinfo)"
+notif() {
+    echo -e "\n$@" \
+            "\n================================================================================" \
+            "\n" >&2
+}
 
+NPROCS="${NPROCS:-$(grep -c '^processor' /proc/cpuinfo)}"
+WORKSPACE="${WORKSPACE:-$(pwd)}"
+
+K_EXERCISES_REPO="${K_EXERCISES_REPO:-git@github.com:kframework/k-exercises.git}"
+RVMATCH_REPO="${RVMATCH_REPO:-git@github.com:runtimeverification/rv-match.git}"
+
+KSERVER_SOCKET="$WORKSPACE/kserver"
+KSERVER_LOG="$WORKSPACE/kserver.log"
+export KSERVER_SOCKET
+
+notif "CLEANING REPO"
+mvn clean
+git clean -dffx
```

Comment:
Also what does `-ff` do? Unless this option cleans submodules then this is still not going to work.

Reply:
From `man git clean`:

```
       -f, --force
           If the Git configuration variable clean.requireForce is not set to false, git clean will
           refuse to delete files or directories unless given -f, -n or -i. Git will refuse to delete
           directories with .git sub directory or file unless a second -f is given.
```

Reply:
So it makes sure that the `rv-match` and `k-exercises` repositories are actually deleted along with any other junk.